frontend/src/components/homepage/Filter.tsx

```diff
@@ -40,43 +51,49 @@ const Filter = ({
       <option key={i.toString()} value={i.toString()} label={i.toString()} />,
     );
   }
+  const filterStyle = useStyleConfig("Filter");
```

Comment:
`useStyleConfig` example

Reply:
Is there a reason we need to use the useStyleConfig hook here instead of just migrating the CSS to Chakra?

Reply:
I moved the css to Chakra (`themes/components/Filter`) for now and load it here through useStyleConfig 
I initially tried to put this css into a theme for `Flex` but `Flex` & `Box` don't accept variants and have to be loaded like this ([source](https://github.com/chakra-ui/chakra-ui/issues/2456))
(for context, the original component with `className="filter"` was a flex `div`, hence the desire to convert it to `Flex`)

Reply:
The unfortunate `sx` usage is also a result of `useStyleConfig`